symfony/plugins/orangehrmAdminPlugin/Api/EducationAPI.php

```diff
@@ -0,0 +1,160 @@
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Admin\Api;
+
+use DaoException;
+use Exception;
+use OrangeHRM\Admin\Api\Model\EducationModel;
+use OrangeHRM\Admin\Dto\UserSearchFilterParams;
+use OrangeHRM\Admin\Service\EducationService;
+use OrangeHRM\Core\Api\V2\CrudEndpoint;
+use OrangeHRM\Core\Api\V2\Endpoint;
+use OrangeHRM\Core\Api\V2\Model\ArrayModel;
+use OrangeHRM\Core\Api\V2\ParameterBag;
+use OrangeHRM\Core\Api\V2\RequestParams;
+use OrangeHRM\Core\Api\V2\Serializer\EndpointCreateResult;
+use OrangeHRM\Core\Api\V2\Serializer\EndpointDeleteResult;
+use OrangeHRM\Core\Api\V2\Serializer\EndpointGetAllResult;
+use OrangeHRM\Core\Api\V2\Serializer\EndpointGetOneResult;
+use OrangeHRM\Core\Api\V2\Serializer\EndpointUpdateResult;
+use OrangeHRM\Entity\Education;
+use Orangehrm\Rest\Api\Exception\RecordNotFoundException;
+
+class EducationAPI extends EndPoint implements CrudEndpoint
+{
+    /**
+     * @var null|EducationService
+     */
+    protected ?EducationService $educationService = null;
+
+    public const PARAMETER_ID = 'id';
+    public const PARAMETER_IDS = 'ids';
+    public const PARAMETER_NAME = 'name';
+
+    /**
+     *
+     * @return EducationService
+     */
+    public function getEducationService(): EducationService
+    {
+        if (is_null($this->educationService)) {
+            $this->educationService = new EducationService();
+        }
+        return $this->educationService;
+    }
+
+    /**
+     * @param EducationService $educationService
+     */
+    public function setEducationService(EducationService $educationService): void
+    {
+        $this->educationService = $educationService;
+    }
+
+    /**
+     * @return EndpointGetOneResult
+     * @throws RecordNotFoundException
+     * @throws DaoException
+     * @throws Exception
+     */
+    public function getOne(): EndpointGetOneResult
+    {
+        // TODO:: Check data group permission
+        $id = $this->getRequestParams()->getInt(RequestParams::PARAM_TYPE_ATTRIBUTE, self::PARAMETER_ID);
+        $educations = $this->getEducationService()->getEducationById($id);
+        if (!$educations instanceof Education) {
```

Comment:
I think you can simplify `if (!$educations)` throw exception since getEducationById returns null